radium-blockchain-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,233 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"adminUsername": {
+			"type": "string",
+			"metadata": {
+				"description": "Username for the Virtual Machine."
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "Password for the Virtual Machine."
+			}
+		},
+		"dnsLabelPrefix": {
+			"type": "string",
+			"metadata": {
+				"description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+			}
+		},
+		"installMethod": {
+			"type": "string",
+			"defaultValue": "From_Source",
+			"allowedValues": [
+				"From_Source"
+			],
+			"metadata": {
+				"description": "Method to install Radium. From_Source: official Radium repo on GitHub."
+			}
+		},
+		"vmSize": {
+			"type": "string",
+			"defaultValue": "Standard_A2",
+			"allowedValues": [
+				"Standard_A1",
+				"Standard_A2",
+				"Standard_A3",
+				"Standard_D1_V2",
+				"Standard_D2_V2",
+				"Standard_D3_V2"
+			],
+			"metadata": {
+				"description": "Size of VM"
+			}
+		},
+		"rpcUser": {
+			"type": "string",
+			"metadata": {
+				"description": "Username for the RPC connection."
+			}
+		},
+		"rpcPass": {
+			"type": "securestring",
+			"metadata": {
+				"description": "Password for the RPC connection."
+			}
+		},
+		"rpcPort": {
+			"type": "string",
+			"metadata": {
+				"description": "Port for the RPC connection."
+			}
+		},
+		"rpcAllowIp": {
+			"type": "string",
+			"metadata": {
+				"description": "Allowed IP for RPC connection."
+			}
+		}
+	},
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "OSDiskName": "RadiumOSDisk",
+    "imageSKU": "15.10",
```

Comment:
Can this work on 14.04 LTS? 15.10 will be EOL soon and will be removed from Azure soon after causing this template to break